modules/gdpr_tasks/src/Entity/Task.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+namespace Drupal\gdpr_tasks\Entity;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Entity\EntityChangedTrait;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\user\UserInterface;
+
+/**
+ * Defines the Task entity.
+ *
+ * @ingroup gdpr_tasks
+ *
+ * @ContentEntityType(
+ *   id = "gdpr_task",
+ *   label = @Translation("Task"),
+ *   bundle_label = @Translation("Task type"),
+ *   label_collection = @Translation("Task list"),
```

Comment:
These probably should have a GDPR prefix (E.g "GDPR Task") for clarity.

Reply:
I've moved it under admin/gdpr menu structure.